### DISCUSSION TITLE
The data cache is not balanced

### DISCUSSION THREAD

**Discussion by liyangbing** [juicedata/juicefs#1613] [General]
use juiceFS sync pull 2.2T data from external OSS to the JuiceFS data directory from a machine
the data cache of the pull machine accounts for almost 2T, and the other 15 machines are hardly used
is that how it works? Or is there something wrong with the Settings
Performance should be limited by disk reads and writes on a single machine

**What you expected to happen**:
Data is evenly distributed to each data cache node

**Environment**:
Juicefs: 1.0.0 - beta 2
Operation : 4.15.0-171- Generic #180-Ubuntu
Metadata: Redis primary and secondary
Number of data caches machine: 16

**Comment by davies** [comment]
The data cache is built separately, not shared with each other, the size of cached data is determined by workload on them. So this is expected.

**Reply by dannyfe** [reply]
Is there a plan to build a shared cache?

**Reply by davies** [reply]
Yes, there is a PR: https://github.com/juicedata/juicefs/pull/1364
